src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -689,6 +690,16 @@ export const duplicateForm = (
 
   const duplicateParams = originalForm.getDuplicateParams(overrideProps)
 
+  // remove conditional routing mapping from dropdown fields
+  if (duplicateParams?.form_fields && duplicateStripped) {
+    duplicateParams.form_fields.forEach((field) => {
+      if (field.fieldType === BasicField.Dropdown) {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        ;(field as any).optionsToRecipientsMap = undefined
```

Comment:
I did a quick check and setting `optionsToRecipientsMap` to undefined directly should work:

```suggestion
        field.optionsToRecipientsMap = undefined
```

<img width="721" height="431" alt="image" src="https://github.com/user-attachments/assets/e3678251-0952-43df-ab69-38efb763d7b7" />

Reply:
Committed this change! Thanks 🙏

Reply:
works! 💯